micrometer-core/src/main/java/io/micrometer/core/instrument/binder/httpcomponents/hc5/MicrometerHttpRequestExecutor.java

```diff
@@ -71,10 +72,10 @@ public class MicrometerHttpRequestExecutor extends HttpRequestExecutor {
     /**
      * Use {@link #builder(MeterRegistry)} to create an instance of this class.
      */
-    private MicrometerHttpRequestExecutor(MeterRegistry registry, Function<HttpRequest, String> uriMapper,
-            Iterable<Tag> extraTags, boolean exportTagsForRoute, ObservationRegistry observationRegistry,
-            @Nullable ApacheHttpClientObservationConvention convention) {
-        super();
+    private MicrometerHttpRequestExecutor(Timeout waitForContinue, MeterRegistry registry,
```

Comment:
`waitForContinue` has been removed from the Apache HTTP client 4 support for some reason. I just revert it back just in case, but please let me know if it was intentional for some reason.

Reply:
I did somewhat intentionally remove it, but I should have mentioned that in my pull request. It wasn't clear to me that users need it. It was included in the original pull request that added support for HC 4 without any mention. I was going to wait until a user asked for it before adding it, but it isn't really a problem to include it for parity with the existing instrumentation for HC 4.